nodeadm/test/e2e/cases/kubeconfig-outpost/run.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source /helpers.sh
+
+mock::imds
+wait::dbus-ready
+
+mock::kubelet 1.28.0
+nodeadm init --skip run --config-source file://config.yaml
+assert::file-contains /etc/hosts 'example.com'
```

Comment:
wait, this is hitting the real `example.com`? can we use `localhost` or mock a hostname and then include the whole `'IP HOST'` string in the assert?